samples/kotlin_compose/src/main/java/io/customer/android/sample/kotlin_compose/navigation/AppNavigation.kt

```diff
@@ -36,10 +39,15 @@ sealed class Screen(val route: String) {
 @Composable
 fun AppNavGraph(
     modifier: Modifier = Modifier,
+    deepLinkState: StateFlow<Intent?>,
     startDestination: String = Screen.Login.route
 ) {
     val navController = rememberNavController()
 
+    deepLinkState.collectAsState(null).value.let {
+        it?.let { navController.handleDeepLink(it) }
+    }
```

Comment:
Can we simplify it as follows?

```suggestion
    deepLinkState.collectAsState(null).value?.let { intent ->
        navController.handleDeepLink(intent)
    }
```